BikeRentalApi/endpoints/stationsDetailBikes.py

```diff
@@ -84,3 +99,18 @@ def post(request, pk):
         safe = False,
         status = status.HTTP_201_CREATED
     )
+
+
+@RoleRequired([Role.Tech, Role.Admin])
+def get_all(request, pk):
```

Comment:
Broken naming convention, but this method is temporary so let's leave it this way for now.